### PR TITLE
9987 kill quadratic lands w migrations

### PR DESCRIPTION
**Pull request by schultztimothy**
Description
As a Gitcoin user
I want to only see the decentralized version of the Quadratic Lands site
So that I am only seeing the correct, community-owned information

Acceptance Criteria
GIVEN that I am on gitcoin.co
WHEN I try to visit https://gitcoin.co/quadraticlands
THEN I am sent to http://decentralized.quadraticlands.com/

we no longer should be maintaining this portion of the site. it is now managed by the DAO per https://gitcoin.co/issue/gitcoinco/skunkworks/239/100026286

Refers/Fixes
fixes: https://github.com/gitcoinco/web/issues/9987

According to https://techstream.org/Bits/Remove-App-From-Django-Project its best to follow this up with another [PR](https://github.com/gitcoinco/web/pull/10138) that fully removes the app from the monorepo. Still pretty new to django so open to suggestions on this 😄

### CODE REVIEW

File: app/dashboard/tasks.py [hunk 1]
```diff
@@ -281,12 +281,6 @@ def update_trust_bonus(self, pk):
         tb += 0.15
     # if profile.is_duniter_verified:
     #     tb *= 1.001
-    qd_tb = 0
```
[the remark this answers is not in the document]
Comment (thelostone-mc) [reply app/dashboard/tasks.py:284]: lgtm

File: app/quadraticlands/migrations/0001_initial.py [hunk 2]
```diff
@@ -40,7 +40,7 @@ class Migration(migrations.Migration):
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('created_on', models.DateTimeField(auto_now=True)),
-                ('claim_total', quadraticlands.models.Uint256Field(default=0)),
+                ('claim_total', models.DateTimeField(auto_now=True)),
                 ('distribution', django.contrib.postgres.fields.jsonb.JSONField(default=dict)),
                 ('profile', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='initial_distribution', to='dashboard.Profile')),
             ],
```
Comment (thelostone-mc) [inline app/quadraticlands/migrations/0001_initial.py:46]: ^ could we undo the changes to the migration file as production already has them

**Review by thelostone-mc — CHANGES_REQUESTED** [review]
I'm good with this approach ! 
Left a comment on the migration update but otherwise LGTM

File: app/quadraticlands/migrations/0001_initial.py (hunk 2 above)
Comment (schultztimothy) [reply app/quadraticlands/migrations/0001_initial.py:45]: Ahhh yeah good call!

When I include the `quadraticlands` model it throws an error and the app won't compile locally. 

I updated it so that it is still a `Uint256Field` but open to suggestions on other ways around that 😄

Comment (thelostone-mc) [reply app/quadraticlands/migrations/0001_initial.py:45]: Hmm the issue with that is -> it's not what is running on the server.
Since we'd en up removing this in a follow up PR 
Just to be on the safer side -> maybe we just keep it as original

Comment (schultztimothy) [reply app/quadraticlands/migrations/0001_initial.py]: Sounds good and makes sense 🚀 I reverted so there are no changes to this file

**Review by thelostone-mc — APPROVED** [review]

File: app/app/views.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+from django.shortcuts import redirect
+
+def redirect_view(request):
+    response = redirect('http://decentralized.quadraticlands.com/')
+    return response
```
Comment (ksolo) [inline app/app/views.py:5]:
```suggestion
    return redirect('https://decentralized.quadraticlands.com/')
```

**Review by ksolo — APPROVED** [review]
one small suggestion (non-blocker).

looks good!

File: app/dashboard/tasks.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (gdixon) [reply app/dashboard/tasks.py:284]: We had it in place for a while but backed it out, there might also be some remnants in `app/assets/v2/js/pages/profile-trust.js` and it looks like you've caught the `app/dashboard/views.py` side of it!! 🚀

**Review by gdixon — APPROVED** [review]
LGTM! I've a left a comment about somewhere that might have been missed, but other than that, looking great! 🚀

File: app/dashboard/tasks.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (thelostone-mc) [reply app/dashboard/tasks.py:284]: Ah nice one ! I'll remove that
